dotcom-rendering/src/components/InteractivesNativePlatformWrapper.importable.tsx

```diff
@@ -0,0 +1,20 @@
+import { NativePlatform } from '@guardian/bridget';
+import { log } from '@guardian/libs';
+import { useEffect } from 'react';
+import { getInteractivesClient } from '../lib/bridgetApi';
+
+export const InteractivesNativePlatformWrapper = () => {
+	useEffect(() => {
+		void getInteractivesClient()
+			.getNativePlatform()
+			.then((platform) =>
+				document.body.classList.add(
+					platform === NativePlatform.ios ? 'ios' : 'android',
+				),
```

Comment:
Minor non-blocking. Would this be equivalent?

```suggestion
				document.body.classList.add(NativePlatform[platform]),
```